docs/quick-install.md

```diff
@@ -1,175 +1,184 @@
-
-To install the Smart Agent on a single Linux host, enter:
+##### Option 1: From the SignalFx UI    
+If you are reading this content from the SignalFx Smart Agent tile in the Integrations page, then simply copy and paste the following code into your command line. (The code within the tile is already populated with your realm and your organization's access token.)
 
 ```sh
-curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
-sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM YOUR_SIGNALFX_API_TOKEN
+& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
 ```
 
-__Windows:__ Ensure that the following dependencies are installed:
+***
 
-[.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+##### Option 2: From the documentation site 
+If you are reading this content from the SignalFx documentation site, then SignalFx recommends that you access the Integrations page in the SignalFx UI to copy the pre-populated installation code.  
 
-[Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+1. Log in to SignalFx and click the **Integrations** tab to open the Integrations page. Look for the SignalFx Smart Agent tile. You can search for it by name, or find it in the **Essential Services** section.
+2. Under **Essential Services**, click **SignalFx Smart Agent**.
+3. Click **Setup**.
+4. Locate the text box for Windows users.
+5. Copy, paste, and run the code in your command line. (The code within the tile is already populated with your realm and your organization's access token.)  
 
-* Get your API\_TOKEN from:  __Organization Settings => Access Token__ tab in the SignalFx application.
+The agent will be installed as a Windows service and will log to the Windows Event Log.
+</p>
+</details>
 
-* Determine YOUR\_SIGNAL\_FX_REALM from: your profile page in the SignalFx web application.
+***
 
-To install the Smart Agent on a single Windows host, enter:
+### Step 2. Confirm your installation
 
-```sh
-& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"};; 
-ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
-```
+1. From the command line, enter the following command to confirm that your Smart Agent is functional:
```

Comment:
I thought this was Linux only at first, but the live doc says this command works on both platforms. (Because I didn't know Windows supported sudo.) For clarity, I'd say "From the Linux or Windows command line,...."

Reply:
But also, my standard is to put what you are trying to do first, so it would be "To confirm..., enter the following command on the Linux or Windows command line."